.github/ISSUE_TEMPLATE/vep.md

```diff
@@ -52,11 +52,16 @@ Link to any enhancement PRs.
 See https://github.com/kubevirt/community/blob/main/design-proposals/feature-lifecycle.md#releases for more context, please include links to relevant PRs
 -->
 
-* [] Alpha (v1.5.0?)
-* [] Beta (v1.6.0?)
-  * [] Additional Beta criteria
-* [] GA (v1.7.0?)
-  * [] Additional GA criteria
+* Alpha release target (x.y):
+* Beta release target (x.y):
+* Stable release target (x.y):
+
+
+* [] Alpha
```

Comment:
Does that mean that y vep, code, docs, bugs is expected for every milestone? like this list is duplicated for alpha/beta/ga?

Reply:
No really, it is listed here to provide guidance on what should be tracked, it is unrealistic that every type will be required per each stage.